Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1634,7 +1634,7 @@ extension BrowserViewController: URLBarDelegate {
 extension BrowserViewController: TabDelegate {
 
     func tab(_ tab: Tab, didCreateWebView webView: WKWebView) {
-        webView.frame = webViewContainer.frame
+        webView.frame = webViewContainer?.frame ?? .init()
```

Comment:
@ecotopian just in case

Reply:
looks great! two safety belts are better than one 😉